.github/workflows/integration.yml

```diff
@@ -141,12 +159,19 @@ jobs:
 
     steps:
       - name: Check out the repository
-        if: github.event_name != 'pull_request_target'
+        if: github.event_name != 'pull_request_target' && github.event_name != 'workflow_dispatch'
```

Comment:
Since there are only three options, can we update this conditional to be stated in the positive, i.e. `if: github.event_name == 'push'`?

Reply:
We could. The problem is that right now different adapter repo versions of this file have a different set of event_names they'll process, so I'll have to read each one carefully to make sure that everything is handled and since I imagine that at some point we'd like to have a common version instead of 5 subtly different versions, it felt more robust to name each one. It's certainly a judgment call.

Reply:
Also if somebody adds an additional event_name, it wouldn't default then and they'd have to remember to update these sections.

Reply:
You've convinced me. I've changed them to `== 'push'`.